.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,19 @@
+name: pre-commit
+on: [push, pull_request]
+env:
+  PYTHON_VERSION: "3.10"
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
+      with:
+        python-version: ${{ env.PYTHON_VERSION }}
+    - name: Bootstrap poetry
```

Comment:
What does this line do? Why do we need poetry?

Reply:
That's botched copy pasting from a different GitHub Action file. That step actually has nothing to do with poetry, I renamed the step to `Install dependencies`.